x-pack/test/api_integration/apis/kibana/stats/stats.js

```diff
@@ -9,26 +9,36 @@ import expect from '@kbn/expect';
 export default function ({ getService }) {
   const supertestNoAuth = getService('supertestWithoutAuth');
   const supertest = getService('supertest');
+  const config = getService('config');
 
   describe('/api/stats', () => {
     describe('operational stats and usage stats', () => {
 
+      // lazy check for uuid for test runs against preexisting services
+      function isUUID(uuid) {
+        return typeof uuid === 'string' && uuid.length === 36;
+      }
+
       describe('no auth', () => {
-        it('should return 200 and stats for no extended', async () => {
+        // depends on kibana.yml setting status.allowAnonymous
+        // skip this test when running against a remote host, as we can't
+        // validate the status of this setting
+        const host = config.get('servers.kibana.host');
```

Comment:
This should be: 
const host = config.get('servers.kibana.hostname');
Otherwise we get a error unknown config key.